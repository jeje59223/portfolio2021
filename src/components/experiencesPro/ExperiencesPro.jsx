import React from 'react';
import '../../styles/experiencesPro.css';
import Card from 'react-bootstrap/Card';
import { Accordion } from 'react-bootstrap';
import data from '../../data/experiencesPro.json';

const ExperiencesPro = () => (
  <section id="experiencesProfessionnelles">
    <div className="container">
      <h2 className="text-center h2ExperiencesProfessionnelles">Expériences Professionnelles</h2>
      <div className="row align-items-center">
        <div className="col-md-6">
          <h2>{data[0].periode}</h2>
          <hr />
          <h4>
            {data[0].entreprise}
            à
            {data[0].ville}
            |
            {data[0].poste}
          </h4>
          <p>
            <span className="fw-bold">Tâches réalisées : </span>
            {data[0].missions}
          </p>
        </div>
        <div className="col-md-6">
          <Accordion defaultActiveKey="1">
            {
              data.map((experience) => (
                <Card key={experience.id}>
                  <Accordion.Toggle as={Card.Header} eventKey={experience.id} className="head">
                    {experience.periode}
                  </Accordion.Toggle>
                  <Accordion.Collapse eventKey={experience.id}>
                    <Card.Body>
                      <h4>
                        {experience.entreprise}
                        à
                        {experience.ville}
                        |
                        {experience.poste}
                      </h4>
                      <p>
                        <span className="fw-bold">Tâches réalisées :</span>
                        {experience.missions}
                      </p>
                    </Card.Body>
                  </Accordion.Collapse>
                </Card>
              ))
            }
          </Accordion>
        </div>
      </div>
    </div>
  </section>
);

export default ExperiencesPro;
