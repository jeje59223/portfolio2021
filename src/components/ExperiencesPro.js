import React from "react";
import "../styles/experiencesPro.css";
import Accordion from "react-bootstrap/Accordion";
import Card from "react-bootstrap/Card";

const ExperiencesPro = () => {
    return (
        <section id="experiencesProfessionnelles">
            <div class="container">
                <h2 class="text-center h2ExperiencesProfessionnelles">Expériences Professionnelles</h2>
                <div class="row align-items-center">
                    <div class="col-md-6">
                        <h2>De 2003 à Aujourd'hui</h2>
                        <hr/>
                        <h4>Leroy Merlin Villeneuve d'Ascq | Conseiller de vente expert </h4>
                        <p>
                            <span class="fw-bold">Tâches réalisées :</span> Fiabilisation des stocks et du taux de rupture. "commandes, relevés de ruptures, 
                            inventaires,....". Suivi des indicateurs pour atteindre les objectifs de ventes. Contact avec les 
                            fournisseurs. Formé pour répondre aux besoin des consommateurs dans un secteur conccurentiel. 
                            Elaboration et suivi de stratégies en équipe "débriefing en réunion secteur". Accueil, formation et 
                            évaluation des nouveaux recrutés. Force de proposition en centrale d'achat (nouveautés, services, etc....).
                        </p>
                    </div>
                    <div class="col-md-6">
                    <Accordion defaultActiveKey="0">
                    <Card>
                        <Accordion.Toggle as={Card.Header} eventKey="0">
                        De Septembre 2020 à Décembre 2020
                        </Accordion.Toggle>
                        <Accordion.Collapse eventKey="0">
                        <Card.Body>
                            <h4>Fewlines à Lille | Bootcamp Dévelopeur Web</h4>
                            <p>
                            <span class="fw-bold">Tâches réalisées :</span> Apprentissage de plusieurs languages tel que HTML, CSS, Bootstrap,
                             JavaScript, NodeJS, React, Redux, React Native et NextJS. Nous avons appris à utiliser Git, Github et comme base de données
                             MongoDB. Nous avons rélisées plusieurs projets durant cette formation qui m'ont permis de réussir ma reconversion Professionnelle
                             et de devenir développeur web junior.
                            </p>
                        </Card.Body>
                        </Accordion.Collapse>
                    </Card>
                    <Card>
                        <Accordion.Toggle as={Card.Header} eventKey="1">
                        De Mars 2003 à Septembre 2020
                        </Accordion.Toggle>
                        <Accordion.Collapse eventKey="1">
                        <Card.Body>
                            <h4>Leroy Merlin Villeneuve d'Ascq | Conseiller de vente expert</h4>
                            <p>
                            <span class="fw-bold">Tâches réalisées :</span> Fiabilisation des stocks et du taux de rupture. "commandes, relevés de ruptures, 
                            inventaires,....". Suivi des indicateurs pour atteindre les objectifs de ventes. Contact avec les 
                            fournisseurs. Formé pour répondre aux besoin des consommateurs dans un secteur conccurentiel. 
                            Elaboration et suivi de stratégies en équipe "débriefing en réunion secteur". Accueil, formation et 
                            évaluation des nouveaux recrutés. Force de proposition en centrale d'achat (nouveautés, services, etc....).
                            </p>
                        </Card.Body>
                        </Accordion.Collapse>
                    </Card>
                    <Card>
                        <Accordion.Toggle as={Card.Header} eventKey="2">
                            De Février à Juillet 2019
                        </Accordion.Toggle>
                        <Accordion.Collapse eventKey="2">
                        <Card.Body>
                            <h4>M2I Formation à Villeneuve d'Ascq | Développeur Concepteur d'Applications</h4>
                            <p>
                                <span class="fw-bold">Tâches réalisées :</span> Apprentissage de plusieurs languages tel que HTML, CSS, JavaScript,
                                PHP, Symfony, Java et Android Studio. Cette formation m'a permis d'obtenir mon certificat professionnel de Développeur Concepteur
                                d'Applications, de niveau II, équivalent à une licence.
                            </p>
                        </Card.Body>
                        </Accordion.Collapse>
                    </Card>
                    <Card>
                        <Accordion.Toggle as={Card.Header} eventKey="3">
                            De Juillet 2019 à Octobre 2019
                        </Accordion.Toggle>
                        <Accordion.Collapse eventKey="3">
                        <Card.Body>
                            <h4>IT Room à Hem | Stage Développeur PHP Symfony </h4>
                            <p>
                                <span class="fw-bold">Tâches réalisées :</span> Développement d'une application permettant la suveillance de l'ensemble 
                                d'un parc applicatif. Maintenance d'application. Développement d'API (ApiPlatform).
                            </p>
                        </Card.Body>
                        </Accordion.Collapse>
                    </Card>
                    </Accordion>
                    </div>
                </div>
            </div>
        </section>
    )
}

export default ExperiencesPro;