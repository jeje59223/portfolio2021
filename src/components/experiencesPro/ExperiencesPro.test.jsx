import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';
import React from "react";
import ExperiencesPro from "./ExperiencesPro";

configure({ adapter: new Adapter() });

describe('<ExperiencesPro>', () => {
  it('should render ExperiencesPro component', () => {
    const wrapper = shallow(<ExperiencesPro />);
    const json = toJson(wrapper);
    expect(json).toMatchSnapshot();
  });
});