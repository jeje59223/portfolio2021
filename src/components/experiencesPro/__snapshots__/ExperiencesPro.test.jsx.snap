// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ExperiencesPro> should render ExperiencesPro component 1`] = `
<section
  id="experiencesProfessionnelles"
>
  <div
    className="container"
  >
    <h2
      className="text-center h2ExperiencesProfessionnelles"
    >
      Expériences Professionnelles
    </h2>
    <div
      className="row align-items-center"
    >
      <div
        className="col-md-6"
      >
        <h2>
          De Décembre 2020 à Aujourd'hui
        </h2>
        <hr />
        <h4>
          Leroy Merlin
           
          à
           
          Lezennes
           
          |
           
          Web Developer
        </h4>
        <p>
          <span
            className="fw-bold"
          >
            Tâches réalisées : 
          </span>
          Développeur Front-End React Junior. Au sein de la Digital & Data Factory, Plateforme des Services, Espace Partenaires (Tandem).
        </p>
      </div>
      <div
        className="col-md-6"
      >
        <Accordion
          defaultActiveKey="1"
        >
          <Card
            body={false}
            key="1"
          >
            <ForwardRef
              as={
                Object {
                  "$$typeof": Symbol(react.forward_ref),
                  "defaultProps": undefined,
                  "render": [Function],
                }
              }
              className="head"
              eventKey="1"
            >
              De Décembre 2020 à Aujourd'hui
            </ForwardRef>
            <AccordionCollapse
              eventKey="1"
            >
              <CardBody>
                <h4>
                  Leroy Merlin
                   
                  à
                   
                  Lezennes
                   
                  |
                   
                  Web Developer
                </h4>
                <p>
                  <span
                    className="fw-bold"
                  >
                    Tâches réalisées :
                  </span>
                  Développeur Front-End React Junior. Au sein de la Digital & Data Factory, Plateforme des Services, Espace Partenaires (Tandem).
                </p>
              </CardBody>
            </AccordionCollapse>
          </Card>
          <Card
            body={false}
            key="2"
          >
            <ForwardRef
              as={
                Object {
                  "$$typeof": Symbol(react.forward_ref),
                  "defaultProps": undefined,
                  "render": [Function],
                }
              }
              className="head"
              eventKey="2"
            >
              De Septembre 2020 à Décembre 2020
            </ForwardRef>
            <AccordionCollapse
              eventKey="2"
            >
              <CardBody>
                <h4>
                  Fewlines
                   
                  à
                   
                  Lille
                   
                  |
                   
                  Bootcamp Dévelopeur Web
                </h4>
                <p>
                  <span
                    className="fw-bold"
                  >
                    Tâches réalisées :
                  </span>
                  Apprentissage de plusieurs languages tel que HTML, CSS, Bootstrap, JavaScript, NodeJS, React, Redux, React Native et NextJS. Nous avons appris à utiliser Git, Github et comme base de données MongoDB. Nous avons rélisées plusieurs projets durant cette formation qui m'ont permis de réussir ma reconversion Professionnelle et de devenir développeur web junior.
                </p>
              </CardBody>
            </AccordionCollapse>
          </Card>
          <Card
            body={false}
            key="3"
          >
            <ForwardRef
              as={
                Object {
                  "$$typeof": Symbol(react.forward_ref),
                  "defaultProps": undefined,
                  "render": [Function],
                }
              }
              className="head"
              eventKey="3"
            >
              De Mars 2003 à Septembre 2020
            </ForwardRef>
            <AccordionCollapse
              eventKey="3"
            >
              <CardBody>
                <h4>
                  Leroy Merlin
                   
                  à
                   
                  Villeneuve d'Ascq
                   
                  |
                   
                  Conseiller de vente expert
                </h4>
                <p>
                  <span
                    className="fw-bold"
                  >
                    Tâches réalisées :
                  </span>
                  Fiabilisation des stocks et du taux de rupture. "commandes, relevés de ruptures, inventaires,....". Suivi des indicateurs pour atteindre les objectifs de ventes. Contact avec les fournisseurs. Formé pour répondre aux besoin des consommateurs dans un secteur conccurentiel. Elaboration et suivi de stratégies en équipe "débriefing en réunion secteur". Accueil, formation et évaluation des nouveaux recrutés. Force de proposition en centrale d'achat (nouveautés, services, etc....).
                </p>
              </CardBody>
            </AccordionCollapse>
          </Card>
          <Card
            body={false}
            key="4"
          >
            <ForwardRef
              as={
                Object {
                  "$$typeof": Symbol(react.forward_ref),
                  "defaultProps": undefined,
                  "render": [Function],
                }
              }
              className="head"
              eventKey="4"
            >
              De Février à Juillet 2019
            </ForwardRef>
            <AccordionCollapse
              eventKey="4"
            >
              <CardBody>
                <h4>
                  M2I Formation
                   
                  à
                   
                  Villeneuve d'Ascq
                   
                  |
                   
                  Développeur Concepteur d'Applications
                </h4>
                <p>
                  <span
                    className="fw-bold"
                  >
                    Tâches réalisées :
                  </span>
                  Apprentissage de plusieurs languages tel que HTML, CSS, JavaScript, PHP, Symfony, Java et Android Studio. Cette formation m'a permis d'obtenir mon certificat professionnel de Développeur Concepteur d'Applications, de niveau II, équivalent à une licence.
                </p>
              </CardBody>
            </AccordionCollapse>
          </Card>
          <Card
            body={false}
            key="5"
          >
            <ForwardRef
              as={
                Object {
                  "$$typeof": Symbol(react.forward_ref),
                  "defaultProps": undefined,
                  "render": [Function],
                }
              }
              className="head"
              eventKey="5"
            >
              De Juillet 2019 à Octobre 2019
            </ForwardRef>
            <AccordionCollapse
              eventKey="5"
            >
              <CardBody>
                <h4>
                  IT Room
                   
                  à
                   
                  Hem
                   
                  |
                   
                  Stage Développeur PHP Symfony
                </h4>
                <p>
                  <span
                    className="fw-bold"
                  >
                    Tâches réalisées :
                  </span>
                  Développement d'une application permettant la suveillance de l'ensemble d'un parc applicatif. Maintenance d'application. Développement d'API (ApiPlatform).
                </p>
              </CardBody>
            </AccordionCollapse>
          </Card>
        </Accordion>
      </div>
    </div>
  </div>
</section>
`;
